gazebo: true # Whether the robot is started in a Gazebo simulation environment
collision_check: true # Check collisions?
command_in_topic:  jog_arm_server/delta_jog_cmds
joint_command_in_topic: jog_arm_server/joint_delta_jog_cmds
command_frame:  base_link  # TF frame that incoming cmds are given in
incoming_command_timeout:  5  # Stop jogging if X seconds elapse without a new cmd
joint_topic:  joint_states
move_group_name:  arm  # Often 'manipulator' or 'arm'
singularity_threshold:  45  # Slow down when the condition number hits this (close to singularity)
hard_stop_singularity_threshold: 50 # Stop when the condition number hits this
command_out_topic:  sia5_controller/command
planning_frame:  base_link  # The MoveIt! planning frame. Often 'base_link'
low_pass_filter_coeff:  2.  # Larger --> trust the filtered data more, trust the measurements less.
publish_period:  0.008  # 1/Nominal publish rate [seconds]
publish_delay: 0.005  # delay between calculation and execution start of command
scale:
  linear:  0.0025  # Max linear velocity. Meters per publish_period. Units is [m/s]
  rotational:  0.005  # Max angular velocity. Rads per publish_period. Units is [rad/s]
  joint: 0.01  # Max joint angular/linear velocity. Rads or Meters per publish period. Units is [rad/s] or [m/s].
# Publish boolean warnings to this topic
warning_topic:  jog_arm_server/warning
joint_limit_margin: -0.1 # added as a buffer to joint limits [radians]. If moving quickly, make this larger.
# Can save some bandwidth as most robots only require positions or velocities
publish_joint_positions: true
publish_joint_velocities: true
